### PR TITLE
some style updates

### PR DESCRIPTION
**Pull request by MelSumner**
- made the max page width consistent with other elements
- made the buttons consistent on the blog and survey page
- fixed the a11y of the button with the image
- put a tinge of color on the body background- the white is just toooooo stark. 
- made the background of the survey reminder the same accent blue used for the "jobs" image and the post author names (we can ditch this if @mixonic doesn't like it)

Screenshot of using the blue on the survey banner: 
image

### CODE REVIEW

**Comment by mixonic** [comment]
@MelSumner Seems great. Blue bar seems fine- pops great on desktop.

The button on the landing page animates nicely on hover: https://ember-website-staging-pr-3204.herokuapp.com/ember-community-survey-2018/

The one on the blog post just gets taller: https://ember-website-staging-pr-3204.herokuapp.com/blog/2018/02/21/2018-ember-community-survey.html

Should they be the same?

File: source/stylesheets/components/buttons.css.scss [hunk 1]
```diff
@@ -65,13 +52,13 @@
   text-transform: uppercase;
   text-shadow: rgba(0, 0, 0, 0.3) 0 1px 0;
 
-  &.orange {
-    background-image: url('/images/small-orange-button.png');
-  }
+  // &.orange {
```
Comment (mixonic) [inline source/stylesheets/components/buttons.css.scss]: commented stuff can probably be removed.

**Comment by MelSumner** [comment]
oh and it turns blue! def not supposed to do that. I'll make sure the hover transitions are the same all the way around.

**Comment by MelSumner** [comment]
Because both the job board banner and the survey banner are using the same background color, and the job board is now included in the site navigation, the job banner is turned off on mobile devices until after EmberConf 2018. Issue #3205 will track this.

**Pull request merged**